doc/manual/syntax.md

```diff
@@ -205,6 +205,14 @@ def concat(str_array, log=false):
 Symbolic strings are another type of special strings in Nickel. For example, you
 could encounter the following code in a library for using Nickel with Nix:
 
+For concrete use-cases, some tools require manipulating string-like values that
+are not yet known at the time of evaluation (e.g. Terraform's computed values),
+or which performs additional dependency tracking ([Nix string
```

Comment:
```suggestion
or they perform additional dependency tracking ([Nix string
```
This sentence gets a bit convoluted, but I think this change makes it more readable

Reply:
Hmm. Maybe I should just break this sentence in two.